packages/datadog-plugin-elasticsearch/test/index.spec.js

```diff
@@ -38,9 +38,8 @@ describe('Plugin', () => {
           agent
             .use(traces => {
               expect(last(traces[0])).to.have.property('resource', 'POST /logstash-?.?.?/_search')
+              done()
```

Comment:
This hides any issues and doesn't actually solve them. Was the source of the problem found?